packages/ui/src/2_molecules/NavMenuItem/NavMenuItem.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC, ReactNode, useMemo, useRef } from 'react';
+
+import classNames from 'classnames';
+
+import { Badge } from '../../1_atoms/Badge/Badge';
+import styles from './NavMenuItem.module.css';
+
+const MAX_NOTIFICATIONS_COUNT = 99;
+
+type NavMenuItemProps = {
+  count?: number;
+  onClick?: () => void;
+  children: ReactNode;
+  isActive?: boolean;
+  className?: string;
+  dataActionId?: string;
+};
+
+export const NavMenuItem: FC<NavMenuItemProps> = ({
+  count,
+  onClick,
+  children,
+  isActive,
+  className,
+  dataActionId,
+}) => {
+  const ref = useRef<HTMLDivElement>(null);
+  const formattedCount = useMemo(
+    () =>
+      count && count > MAX_NOTIFICATIONS_COUNT
+        ? MAX_NOTIFICATIONS_COUNT
```

Comment:
if it's more than `99`, it should return `99+`

```suggestion
        ? `${MAX_NOTIFICATIONS_COUNT}+`
```